benchmarks/dynamo/common.py

```diff
@@ -2629,6 +2629,13 @@ def get_example_inputs(self):
         default=1,
         help="Set per-process GPU memory fraction (limit) for reducing usable size and reproducing OOMs",
     )
+
+    parser.add_argument(
+        "--no-translation-validation",
+        action="store_true",
+        help="Disable translation validation for accuracy builds.",
+    )
```

Comment:
Had to change this in order for it to work with Python 3.8, since, on that version, there's no `argparse.BooleanOptionalAction`.